internal/wal_receive_handler.go

```diff
@@ -105,6 +107,7 @@ func HandleWALReceive(uploader *WalUploader) {
 			tracelog.ErrorLogger.FatalOnError(err)
 			err = uploader.UploadWalFile(ioextensions.NewNamedReaderImpl(tlh, tlh.Name()))
 			tracelog.ErrorLogger.FatalOnError(err)
+			uploadBulkMetadata(uploader, filepath.Join(getWalFolderPath(), segment.Name()))
```

Comment:
Why not move this to the wal_uploader.go too?

Reply:
Moved that logic to wal_uploader.go for the wal-receive command. Thanks